src/Microsoft.ML.Data/Transforms/ConvertTransform.cs

```diff
@@ -4,36 +4,58 @@
 
 #pragma warning disable 420 // volatile with Interlocked.CompareExchange
 
-using Float = System.Single;
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+using Microsoft.ML.Core.Data;
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.CommandLine;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.Data.Conversion;
+using Microsoft.ML.Runtime.EntryPoints;
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.Runtime.Model;
 using Microsoft.ML.Runtime.Model.Onnx;
-using Microsoft.ML.Runtime.Command;
-using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
 
-[assembly: LoadableClass(ConvertTransform.Summary, typeof(ConvertTransform), typeof(ConvertTransform.Arguments), typeof(SignatureDataTransform),
+[assembly: LoadableClass(ConvertTransform.Summary, typeof(IDataTransform), typeof(ConvertTransform), typeof(ConvertTransform.Arguments), typeof(SignatureDataTransform),
     ConvertTransform.UserName, ConvertTransform.ShortName, "ConvertTransform", DocName = "transform/ConvertTransform.md")]
 
-[assembly: LoadableClass(ConvertTransform.Summary, typeof(ConvertTransform), null, typeof(SignatureLoadDataTransform),
+[assembly: LoadableClass(ConvertTransform.Summary, typeof(IDataTransform), typeof(ConvertTransform), null, typeof(SignatureLoadDataTransform),
     ConvertTransform.UserName, ConvertTransform.LoaderSignature, ConvertTransform.LoaderSignatureOld)]
 
-[assembly: LoadableClass(typeof(ConvertTransform.TypeInfoCommand), typeof(ConvertTransform.TypeInfoCommand.Arguments), typeof(SignatureCommand),
```

Comment:
>ConvertTransform.TypeInfoComman [](start = 32, length = 31)

just checking: does this need to stay since you are introducing the command?

Reply:
I moved command to Commands folder, and this assembly attribute moved to that file as well.

---
In reply to: [229178695](https://github.com/dotnet/machinelearning/pull/1439#discussion_r229178695) [](ancestors = 229178695)